### PR TITLE
Polkadot draft schedule

### PR DESCRIPTION
**Pull request by Overkillus**
Scheduled some items but very open to moving/adding/removing things! If there's something missing feel free to raise it.

### CODE REVIEW

File: syllabus/7-Polkadot/README.md [hunk 1]
```diff
@@ -2,22 +2,66 @@
...
+- Asynchronous Backing [Bradley]
+- XCMP [? > Maciej]
+
 ## Day 2
 
 ### Morning
 
+- Light Clients [Nikos]
+- NPoS & Staking [Ankan > Kian > Maciej]
+- Zombienet [Javier > Nikos > Maciej]
```
Comment (wirednkod) [inline syllabus/7-Polkadot/README.md:56]: This is planned to be delivered on 11th of June

File: syllabus/7-Polkadot/README.md [hunk 2]
```diff
@@ -2,22 +2,66 @@
...
+- Coretime [Maciej]
+- Cumulus [Bradley]
+- Asynchronous Backing [Bradley]
+- XCMP [? > Maciej]
+
 ## Day 2
 
 ### Morning
 
+- Light Clients [Nikos]
```
Comment (wirednkod) [inline syllabus/7-Polkadot/README.md:54]: Also this need to be in afternoon - due to difference of time

File: syllabus/7-Polkadot/README.md [hunk 3]
```diff
@@ -2,22 +2,66 @@
...
+    * Fellowship
+* Tooling
+    * What extra tools are available to build on Polkadot?
+    * Cumulus
+    * Zombienet
+
 ## Hands on activities
 
-## Schedule
+- Build a parachain (maybe)
```
Comment (kianenigma) [inline syllabus/7-Polkadot/README.md:28]: It is best to movet these practical parts into polkadot-sdk module now. Would that work?

File: syllabus/7-Polkadot/README.md [hunk 4]
```diff
@@ -2,22 +2,66 @@
...
 
+- Introduction to Polkadot part 2 [Maciej]
+- Data Preparation - Collation & Backing [Maciej]
+- Data Sharding - Data Availability [Bradley]
+- Execution Sharding - Approvals and Disputes [Maciej]
+
 ### Afternoon
 
+- Coretime [Maciej]
+- Cumulus [Bradley]
```
Comment (kianenigma) [inline syllabus/7-Polkadot/README.md:46]:
```suggestion
- Cumulus Deep Dive [Bradley]
```

(noting that an intro to cumulus should come much sooner)

File: syllabus/7-Polkadot/README.md [hunk 5]
```diff
@@ -2,22 +2,66 @@
...
+- Asynchronous Backing [Bradley]
+- XCMP [? > Maciej]
+
 ## Day 2
 
 ### Morning
 
+- Light Clients [Nikos]
+- NPoS & Staking [Ankan > Kian > Maciej]
```
Comment (kianenigma) [inline syllabus/7-Polkadot/README.md:55]: I have an old lecture on this that still holds, and would be glad to re-lecture it. Ankan will be able to mirror if need be as well, so I you can be confident that we can take care of this if it ends up being in the schedule.

File: syllabus/7-Polkadot/README.md (hunk 4 above)
Comment (Overkillus) [reply syllabus/7-Polkadot/README.md:46]: This will be cumulus deep dive. I hope intro takes place in sdk

**Comment by wirednkod** [comment]
@Overkillus Is there any update any this?

**Review by wirednkod — APPROVED** [review]

**Pull request merged**